parastell/invessel_build.py

```diff
@@ -133,15 +133,26 @@ def angles_to_xyz(self, toroidal_angle, poloidal_angles, s, scale):
 
 class RibBasedSurface(ReferenceSurface):
     """An object that uses closed loops of R, Z points (ribs) on planes of
-    constant toroidal angle to approximate the innermost surface of an in
+    constant toroidal angle to approximate the first wall surface of an in-
     vessel build. This class must be used with split_chamber = False.
 
     Arguments:
         rib_data (numpy array): NxMx2 array of of R, Z points. The first
             dimension corresponds to the plane of constant toroidal angle on
             which the closed loop of points lies. The second dimension is the
             location on the closed loop at which the point lies, and the third
-            dimension is the R, Z values of that point.
+            dimension is the R, Z values of that point. ParaStell expects the
+            following from this data set:
+            - The data spans exactly one field period
+            - The coordinates of each toroidal slice (rib) precess counter-
+              clockwise
```

Comment:
I'm not sure that "counter-clockwise" is well defined here.  It depends whether you are looking "forward" or "backward" along the magnetic axis...  I think we want them to precess upwards at the outboard and downwards at the inboard?